integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiSample.java

```diff
@@ -355,6 +355,201 @@ public void testFmwUpdate4UseCase() {
     execTestScriptAndAssertSuccess(DomainType.JRF,"-update4", "Update4 use case failed");
   }
 
+  /**
+   * Test to verify MII sample WLS initial use case using common mount.
+   * Builds image required for the initial use case, creates secrets, and
+   * creates domain resource.
+   * Verifies all WebLogic Server pods are ready, are at the expected
+   * restartVersion, and have the expected image.
+   * Verifies the sample application is running
+   * (response includes "Hello World!").
+   */
+  @Test
```

Comment:
IMO, this seems like quite a bit of cut-and-paste code - which makes a little error prone and harder to maintain/understand.

Could we instead design the ItMiiSample java test to be modal and then run the same test twice, once the 'old way' and once in 'cm mode'?  For example, would it be possible to have a ItMIISampleCM.java that inherits from ItMIISample.java - where the child class sets a boolean to flag to indicate to the parent class that it should add `envMap("DO_CM", "true")`?

Reply:
If that's not possible, then can the content of tests 2 through 11 be moved to helper functions? If so, then the old tests 2 through 11 and the new CM tests 12 through 21 could then share the same helper functions.

Reply:
We have not used inheritance in integration tests so far as both Mark and Ryan suggested not to use in IT's in the past. I can do the helper functions.